shared/constants/fs.js

```diff
@@ -448,7 +448,7 @@ export const folderToFavoriteItems = (
 }
 
 export const viewTypeFromMimeType = (mimeType: string): Types.FileViewType => {
-  if (mimeType.startsWith('text/')) {
+  if (mimeType === 'text/plain' || mimeType.startsWith('text/plain;')) {
```

Comment:
Mime types can include multiple clauses? Is `;` the delimiter? Some quick googling shows `,`. Also, this will only work if `text/plain` is the first entry.

Reply:
The one it broke was `text/plain; charset=utf-8`. Not sure if this whole part is mime type, or only part of `Content-Type` header. I'll find out and if it's latter I'll change the mimetype to only include stuff before `;`.

I think it's fine if this doesn't work for multiple mime types. It'll just show the "default view" which allows download/share.

Reply:
Yeah I think it's part of `Content-Type`. I'll move it to proper place in this PR.

Reply:
Maybe that's why the Share functionality isn't showing the right apps.

Reply:
I think we should be stripping that.

Reply:
Yeah my latest commit does that. PTAL!